src/hooks/useOnLastOrderIndexed.ts

```diff
@@ -8,7 +8,7 @@ import { useAppSelector } from '@/state/appTypes';
  * @param { callback: () => void }
  */
 export const useOnLastOrderIndexed = ({ callback }: { callback: () => void }) => {
-  const [unIndexedClientId, setUnIndexedClientId] = useState<number | undefined>();
+  const [unIndexedClientId, setUnIndexedClientId] = useState<string | undefined>();
```

Comment:
i know you didnt write this but if you're using `useState()` syntax you dont need to specify the `undefined` type there

you can just do `useState<string>()`